README.md

```diff
@@ -194,6 +194,46 @@ az deployment group create --resource-group $RESOURCE_GROUP_NAME --template-file
 
     
     ![A screenshot of the chat app.](./media/web-unstructureddata.png)
+### Running the sample using the Azure Developer CLI (azd)
+
+The Azure Developer CLI (`azd`) is a developer-centric command-line interface (CLI) tool for creating Azure applications.
+
+You need to install it before running and deploying with the Azure Developer CLI. (If you use the devcontainer, everything is already installed)
+
+### Windows
+
+```powershell
+powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
+```
+
+### Linux/MacOS
+
+```
+curl -fsSL https://aka.ms/install-azd.sh | bash
+```
+
+After logging in with the following command, you will be able to use the `azd` cli to quickly provision and deploy the application.
+
+```
+azd auth login
+```
+
+Then, execute the `azd init` command to initialize the environment (You do not need to run this command if you already have the code or have opened this in a Codespace or DevContainer).
+```
+azd init -t chat-with-your-data-solution-accelerator
+```
+Enter an environment name.
+
+**Notes:** the default auth type uses keys, if you want to switch to rbac, please run `azd env set AUTH_TYPE rbac`.
+
+Then, run `azd up` to provision all the resources to Azure and deploy the code to those resources.
+```
+azd up 
+```
+
+Select your desired `subscription` and `location`. Wait a moment for the resource deployment to complete, click the website endpoint and you will see the web app page.
```

Comment:
@jongio - Three services are currently used. In local deployment, we also noticed that packaging in `python` and `npm` takes a lot of time. In fact, `adminweb` and `function` services use the same `requirements.txt` file, but when using azd to deploy, because two independent services need to install packages twice, the deployment time is too long. Do you have any suggestions for optimizing deployment time?

Reply:
Could you have a step that installs the requirements first and then uses a multi-stage docker build step that shares the locations of the packages?

The devcontainer installs all of these requirements as a PostCreate step so as long as we use a convention we can fix it for this too

Reply:
> Could you have a step that installs the requirements first and then uses a multi-stage docker build step that shares the locations of the packages?

@ross-p-smith - If I understand correctly, you mean to install the required `requirements` in a container first, and then use docker to obtain them in other containers where they are needed, right?

@jongio - Do you have any ideas about the above requirements? As far as I know, when using `azd deploy`, the `host=appservice` with `language=py` will first find the requirements in the current path and install it. And using `azd`, the process from provisioning resources to deploying resources is consistent. If we want to pull other images in appservice, it seems that we can only do some operations through the `appCommandLine` parameter during provisioning (docker has been installed in appservice). What do you think about this? Or do you have better optimization suggestions?